__tests__/e2e/import-library/import-library.test.ts

```diff
@@ -0,0 +1,12 @@
+// @ts-ignore
+import { ContentfulClient } from '@onebeyond/agnostic-cms-harmonizer';
```

Comment:
If the TS module import is not working but in fact the code is correct I think there might be a `d.t.s` file declaring the module exports inside the lib.

Reply:
I've removed the ts ignore and it works

Reply:
brilliant!